doc/classes/RenderingServer.xml

```diff
@@ -4727,69 +4727,72 @@
 		<constant name="VIEWPORT_DEBUG_DRAW_NORMAL_BUFFER" value="5" enum="ViewportDebugDraw">
 			Normal buffer is drawn instead of regular scene so you can see the per-pixel normals that will be used by post-processing effects.
 		</constant>
-		<constant name="VIEWPORT_DEBUG_DRAW_VOXEL_GI_ALBEDO" value="6" enum="ViewportDebugDraw">
+		<constant name="VIEWPORT_DEBUG_DRAW_DEPTH_BUFFER" value="6" enum="ViewportDebugDraw">
```

Comment:
Does this have to be added at this position or can it be added at the end? This changes a lot of values that affects extensions and binary compatibility AFAIK so would be nice to be as non-disruptive as possible

Reply:
Moved enums values to the end as to not cause any conflicts.